python/py-readme_renderer/Portfile

```diff
@@ -2,32 +2,43 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
-PortGroup           github 1.0
-
-github.setup        pypa readme_renderer 26.0
 
 name                py-readme_renderer
-platforms           darwin
+version             30.0
+revision            0
 supported_archs     noarch
 license             apache
```

Comment:
not your change @essandess , but this isn't a valid license description, should be `Apache-2` instead? Would you mind changing that at this time as well?

Reply:
Thanks, done.